src/containers/couresCarousel/styles.module.scss

```diff
@@ -21,10 +28,18 @@
     overflow: hidden;
     outline: none;
     height: 100%;
+    
+    @media screen and (max-width: $width-md) {
+      display: none;
+    }
   }
```

Comment:
You hide arrows on mobile version, so now it is hard to understand, that this is a carousel, it looks like a single image.